op-guide/tune-tikv.md

```diff
@@ -89,7 +89,7 @@ sync-log = true
 region-max-size = "384MB"
 # The threshold value of Region split
 region-split-size = "256MB"
-# When the data size in a Region is larger than the threshold value, TiKV checks whether this Region needs split.
+# When the data size change in a Region is larger than the threshold value, TiKV checks whether this Region needs split.
```

Comment:
I don't think the semantic is correct here. It isn't "change is larger...".

Reply:
How about
When the data size in a Region is changed to be larger than the threshold value, TiKV checks whether this Region needs split.

Reply:
Oh, my fault. This comment is for the config below. So it is indeed "change is larger".